src/db/contractSubscriptions/deleteContractSubscription.ts

```diff
@@ -0,0 +1,20 @@
+import { prisma } from "../client";
+
+interface RemoveContractSubscriptionParams {
+  chainId: number;
+  contractAddress: string;
+}
+
+export const deleteContractSubscription = async ({
+  chainId,
+  contractAddress,
+}: RemoveContractSubscriptionParams) => {
+  return prisma.contractSubscriptions.delete({
```

Comment:
should we delete the data from DB or keep a list of history & mark it as deleted on DB level? this, will help users to check what they have used before.

Reply:
we can mark inactive with deletedAt field